Eevee.Sleep.Bot/Workers/OfficialSiteAnnouncementUpdateWatchingWorker.cs

```diff
@@ -0,0 +1,55 @@
+using Discord.WebSocket;
+using Eevee.Sleep.Bot.Controllers.Mongo;
+using Eevee.Sleep.Bot.Exceptions;
+using Eevee.Sleep.Bot.Extensions;
+using Eevee.Sleep.Bot.Models.InGameAnnouncement.OfficialSite;
+using Eevee.Sleep.Bot.Utils.DiscordMessageMaker;
+using Eevee.Sleep.Bot.Workers.Crawlers;
+using MongoDB.Driver;
+
+namespace Eevee.Sleep.Bot.Workers;
+
+public class OfficialSiteAnnouncementUpdateWatchingWorker(
```

Comment:
Make a base class for announcement update watcher, then extend from them
Also make a folder `AnnouncementUpdateWatcher` and put both `OfficialSite` and `InGame` in it